js/localization/messages/zh-CN.json

```diff
@@ -0,0 +1,264 @@
+    "dxDataGrid-summaryMax": "最大值: {0}",
+    "dxDataGrid-summaryMaxOtherColumn": "{1} 的最大值是 {0}",
+    "dxDataGrid-summaryAvg": "平均值: {0}",
+    "dxDataGrid-summaryAvgOtherColumn": "{1} 的平均值是 {0}",
+    "dxDataGrid-summarySum": "总和: {0}",
+    "dxDataGrid-summarySumOtherColumn": "{1} 的总和是 {0}",
+    "dxDataGrid-summaryCount": "计数: {0}",
+    "dxDataGrid-columnFixingFix": "固定",
+    "dxDataGrid-columnFixingUnfix": "不固定",
+    "dxDataGrid-columnFixingLeftPosition": "向左",
+    "dxDataGrid-columnFixingRightPosition": "向右",
+    "dxDataGrid-exportTo": "导出",
+    "dxDataGrid-exportToExcel": "导出Excel文件",
+    "dxDataGrid-excelFormat": "导出文件",
+    "dxDataGrid-selectedRows": "选中行",
+    "dxDataGrid-exportSelectedRows": "导出选中行",
+    "dxDataGrid-exportAll": "导出所有数据",
+    "dxDataGrid-headerFilterEmptyValue": "(空白)",
+    "dxDataGrid-headerFilterOK": "好",
+    "dxDataGrid-headerFilterCancel": "取消",
+    "dxDataGrid-ariaColumn": "列",
+    "dxDataGrid-ariaValue": "值",
+    "dxDataGrid-ariaFilterCell": "过滤单元",
+    "dxDataGrid-ariaCollapse": "折叠",
+    "dxDataGrid-ariaExpand": "展开",
+    "dxDataGrid-ariaDataGrid": "数据网格",
+    "dxDataGrid-ariaSearchInGrid": "在数据网格中搜索",
+    "dxDataGrid-ariaSelectAll": "全选",
+    "dxDataGrid-ariaSelectRow": "选择行",
+
+    "dxTreeList-ariaTreeList": "树列表",
+    "dxTreeList-editingAddRowToNode": "添加",
+
+    "dxPager-infoText": "第{0}页,共{1}页 ({2} 项目)",
+    "dxPager-pagesCountText": "到",
+
+    "dxPivotGrid-grandTotal": "总计",
+    "dxPivotGrid-total": "{0} 总计",
+    "dxPivotGrid-fieldChooserTitle": "字段选择器",
+    "dxPivotGrid-showFieldChooser": "显示字段选择器",
+    "dxPivotGrid-expandAll": "全部展开",
+    "dxPivotGrid-collapseAll": "全部折叠",
+    "dxPivotGrid-sortColumnBySummary": "按 \"{0}\" 列排序",
+    "dxPivotGrid-sortRowBySummary": "按 \"{0}\" 行排序",
+    "dxPivotGrid-removeAllSorting": "清除所有排序",
+    "dxPivotGrid-dataNotAvailable": "不适用",
+    "dxPivotGrid-rowFields": "行字段",
+    "dxPivotGrid-columnFields": "列字段",
+    "dxPivotGrid-dataFields": "数据字段",
+    "dxPivotGrid-filterFields": "过滤字段",
+    "dxPivotGrid-allFields": "所有字段",
+    "dxPivotGrid-columnFieldArea": "将列拖动到此处",
+    "dxPivotGrid-dataFieldArea": "将数据拖动到此处",
+    "dxPivotGrid-rowFieldArea": "将行字段拖到到此次",
+    "dxPivotGrid-filterFieldArea": "拖动筛选字段到此处",
+
+    "dxScheduler-editorLabelTitle": "标题",
+    "dxScheduler-editorLabelStartDate": "开始日期",
+    "dxScheduler-editorLabelEndDate": "结束日期",
+    "dxScheduler-editorLabelDescription": "描述",
+    "dxScheduler-editorLabelRecurrence": "重复",
+
+    "dxScheduler-openAppointment": "打开日程",
+
+    "dxScheduler-recurrenceNever": "从不",
+    "dxScheduler-recurrenceDaily": "日常",
+    "dxScheduler-recurrenceWeekly": "每周",
+    "dxScheduler-recurrenceMonthly": "每月",
+    "dxScheduler-recurrenceYearly": "每年",
+
+    "dxScheduler-recurrenceEvery": "一切",
+    "dxScheduler-recurrenceEnd": "结束重复",
+    "dxScheduler-recurrenceAfter": "之后",
+    "dxScheduler-recurrenceOn": "在",
+
+    "dxScheduler-recurrenceRepeatDaily": "日(s)",
```

Comment:
(s) here is to say "day or days". It seems it's unnecessary for Chinese localization (the same are about other recurrenceRepeat constants).

Reply:
Yes, it is unnecessary in chinese, I just keep it for clearity, when integreated in dx official localization message, `(s)` should be removed.